packages/react-components/src/shared/tests/typescript/arrayify.ts

```diff
@@ -0,0 +1,16 @@
+import { arrayify } from "../../src";
+import { expectAssignable } from "../../../../../../typescript/tests/helpers";
```

Comment:
Could you add an alias for this? maybe a "@typescript" alias and then import from "@typescript/tests"